diff --git a/components/lwip/CMakeLists.txt b/components/lwip/CMakeLists.txt
index 671da81..563ac09 100644
--- a/components/lwip/CMakeLists.txt
+++ b/components/lwip/CMakeLists.txt
@@ -167,6 +167,7 @@ idf_component_register(SRCS "${srcs}"
 # lots of LWIP source files evaluate macros that check address of stack variables
 target_compile_options(${COMPONENT_LIB} PRIVATE -Wno-address)
 target_compile_definitions(${COMPONENT_LIB} PRIVATE ESP_LWIP_COMPONENT_BUILD)
+target_compile_options(${COMPONENT_LIB} PRIVATE -DSNTP_COMP_ROUNDTRIP=1 -DSNTP_CHECK_RESPONSE=2)
 
 set_source_files_properties(
     lwip/src/netif/ppp/pppos.c
